src/magento-payments/onboarding-payments.md

```diff
@@ -1,3 +1,129 @@
 ---
 title: Onboarding Payments
----
\ No newline at end of file
+---
+
+Magento Payments is your turnkey self-service solution---including sandbox testing, fraud prevention, and a super simple setup---for providing robust and secure payment processing for your Magento store.
+
+To get started using Magento Payments you must complete a few onboarding steps to connect your instance with our payments functionality.
+
+1. [Connect your Magento instance](#configure-magento-services) to Magento Software as a Service (SaaS) services. This only needs to be completed once per Magento instance.
+1. [Onboard and set up the sandbox service](#enable-sandbox-testing) (or, alternatively, proceed to [enabling live payments](#enable-live-payments) if you've tested functionality in another environment) to set up a test PayPal payment processing account.
+1. [Enable Magento Payments as your payment method](#set-magento-payments-as-payment-method), in sandbox mode, to start processing test payments in your store.
+1. [Complete Merchant onboarding](#enable-live-payments) to enable live payments for your Magento store(s).
+1. [Activate Magento Payments in live mode](#activate-live-payments) to begin processing live payments in your Magento store.
+
+{:.bs-callout-info}
+If you do not configure your Magento Services in the Admin (step 1 above) you cannot set up sandbox or live payments.
+
+## Onboarding flow
+
+![Onboarding flow]({% link magento-payments/assets/onboardingflow-docs.png %}){: .zoom}
```

Comment:
We should probably clarify the part where the merchant is waiting for approval on the diagram. Maybe at least mention that it doesn't take a lot of time or the page might need to be refreshed (I think we will have to see what the experience will be).

Reply:
We do. In both the Sandbox onboarding and Live payment onboarding we have sections for that.
![Screen Shot 2021-02-10 at 2 32 10 PM](https://user-images.githubusercontent.com/14031342/107568568-cf378000-6bac-11eb-8e5d-de67409519d6.png)

Reply:
I was wondering if we should mention in the documentation what do we expect from the merchant, for example how long the merchant has to wait. Like I said, we can see what the experience will be after we have it developed and come back to this.